NickvisionMoney.Shared/Resources/Strings.nl.resx

```diff
@@ -0,0 +1,677 @@
+		<value>Transacties die geen deel uitmaken van een groep</value>
+	</data>
+		
+	<data name="Imported" xml:space="preserve">
+		<value>Er is {0} transactie geïmporteerd.</value>
+	</data>
+
+	<data name="Imported.Plural" xml:space="preserve">
+		<value>Er zijn {0} transacties geïmporteerd.</value>
+	</data>
+
+	<data name="UnableToImport" xml:space="preserve">
+		<value>Er kan geen informatie worden geïmporteerd omdat het bestandstype niet ondersteund wordt.</value>
+	</data>
+
+	<data name="Exported" xml:space="preserve">
+		<value>De rekening is geëxporteerd naar een bestand.</value>
+	</data>
+
+	<data name="UnableToExport" xml:space="preserve">
+		<value>De rekening kan niet worden geëxporteerd.</value>
+	</data>
+
+	<data name="Transfer.To" xml:space="preserve">
+		<value>Overmaken naar {0}</value>
+	</data>
+
+	<data name="Transfer.From" xml:space="preserve">
+		<value>Overmaken van {0}</value>
+	</data>
+
+	<!--TransactionRow-->
+	<data name="Edit.TransactionRow" xml:space="preserve">
+		<value>Transactie bewerken</value>
+	</data>
+
+	<data name="Delete.TransactionRow" xml:space="preserve">
+		<value>Transactie verwijderen</value>
+	</data>
+
+	<!--GroupRow-->
+	<data name="Edit.GroupRow" xml:space="preserve">
+		<value>Groep bewerken</value>
+	</data>
+
+	<data name="Delete.GroupRow" xml:space="preserve">
+		<value>Groep verwijderen</value>
+	</data>
+
+	<!--GroupDialog-->
+	<data name="Group" xml:space="preserve">
+		<value>Groep</value>
+	</data>
+
+	<!--TransactionDialog-->
+	<data name="Transaction" xml:space="preserve">
+		<value>Transactie</value>
+	</data>
+
+	<data name="RepeatInterval.Never" xml:space="preserve">
+		<value>Nooit</value>
+	</data>
+
+	<data name="RepeatInterval.Daily" xml:space="preserve">
+		<value>Iedere dag</value>
+	</data>
+
+	<data name="RepeatInterval.Weekly" xml:space="preserve">
+		<value>Iedere week</value>
+	</data>
+
+	<data name="RepeatInterval.Monthly" xml:space="preserve">
+		<value>Iedere maand</value>
+	</data>
+
+	<data name="RepeatInterval.Quarterly" xml:space="preserve">
+		<value>Ieder kwartaal</value>
+	</data>
+
+	<data name="RepeatInterval.Yearly" xml:space="preserve">
+		<value>Ieder jaar</value>
+	</data>
+
+	<data name="RepeatInterval.Biyearly" xml:space="preserve">
+		<value>Iedere twee jaar</value>
+	</data>
+
+	<data name="Manage" xml:space="preserve">
+		<value>Beheren</value>
+	</data>
+
+	<data name="View" xml:space="preserve">
+		<value>Bekijken</value>
+	</data>
+
+	<data name="Delete" xml:space="preserve">
+		<value>Verwijderen</value>
+	</data>
+
+	<data name="Upload" xml:space="preserve">
+		<value>Uploaden</value>
+	</data>
+
+	<!--TransferDialog-->
+	<data name="Transfer" xml:space="preserve">
+		<value>Overboeken</value>
+	</data>
+
+	<data name="TransferDescription" xml:space="preserve">
+		<value>TDoor geld over te maken wordt het bedrag in kwestie als uitgave op de oorspronkelijke rekening genoteerd en als inkomst op de andere.</value>
```

Comment:
A typo in the beginning?

Reply:
Definitely. Thanks for pointing everything out - it's fixed now!